src/main/java/Parser.java

```diff
@@ -25,6 +25,9 @@ public static void scan(TaskList taskList, Storage storage ) {
                     int taskNum = Integer.parseInt(inputArr[1]);
                     taskList.taskDone(taskNum);
                     storage.updateFile(taskList);
+                } else if (input.contains("find")) {
+                    String[] inputArr = input.split(" ", 2);
```

Comment:
Still, you are using a number instead of any defined variables at the top of your class.